tools/templates/tools/usertool_filter.jinja

```diff
@@ -0,0 +1,48 @@
+{% extends "layouts/base.jinja" %}
+{% from "macros/badges.jinja" import taxonomy_list, availability_badge %}
+{% from "partials/pagination.jinja" import pagination %}
+
+{% block head_title %}
+    {{ _("Tools") }}
+{% endblock %}
+
+{% block content %}
+
+<div class="row">
+    <div class="col-md-3">
+        <h3>Filters</h3>
+        {{ crispy(filter.form) }}
+    </div>
```

Comment:
Add a link to `Create tool` somewhere on this page. Couldn't this view and template be modified to handle the `Your Tools` interface as well? maybe via an arg passed into url declaration.